os/arch/arm/src/imxrt/imxrt_mtd_flash.c

```diff
@@ -582,8 +573,7 @@ static int imxrt_ioctl(FAR struct mtd_dev_s *dev, int cmd, unsigned long arg)
 
 	case MTDIOC_BULKERASE: {
 		/* Erase the entire device */
-		fdbg("[pid %d]: checkpoint line %d, arg = %d\n", getpid(), __LINE__, arg);
-		ret = imxrt_bulkerase(priv, arg);
+		ret = imxrt_bulkerase(priv, 0);
```

Comment:
We'd better remove 2nd parameter

Reply:
Strictly speaking, MTDIOC_BULKERASE is not used now to run FS TCs successfully.
So I reverted this function, this will be implemented in another PR.